algorithm_catalog/eurac_pv_farm_detection.json

```diff
@@ -82,7 +82,7 @@
       "rel": "openeo-process",
       "type": "application/json",
       "title": "openEO Process Definition",
-      "href": "https://raw.githubusercontent.com/ESA-APEx/apex_algorithms/refs/heads/hv_pv_detection/openeo_udp/eurac_pv_farm_detection/eurac_pv_farm_detection.json"
+      "href": "https://raw.githubusercontent.com/ESA-APEx/apex_algorithms/4003046e3b79ec3ab8dace888a231655db389d66/openeo_udp/eurac_pv_farm_detection/udf_eurac_pvfarm_onnx.py"
```

Comment:
sorry for being vague earlier, I just gave an example permalink, I think you still want the JSON here (you changed it to `.py`)

Reply:
I think you only have to replace the `refs/heads/hv_pv_detection` part from the URL to `4003046e3b79ec3ab8dace888a231655db389d66` (which is the commit you want to point to)